pattern-library/js/select-replace.js

```diff
@@ -60,16 +57,21 @@ define([
 
                     $el.replaceWith([
                         '<div class="' + variables.wrapperClass + '">',
-                            '<select class="' + replaced[0].className + ' is-replaced" id="' + replaced[0].id + '" name="' + replaced[0].name + '" ' + disabled + '>' + replaced[0].innerHTML + '</select>',
-                            '<span class="' + variables.customClass + ' ' + statuses.join(' ') + '" aria-hidden="true">',
-                                '<span class="' + variables.valueClass + '">' + CustomSelectReplacement.setInitialText($el) + '</span>',
-                                '<span class="icon-fallback-glyph">',
-                                    '<span class="icon ' + variables.iconClass + '" aria-hidden="true"></span>',
-                                    '<span class="text">Down arrow</span>',
-                                '</span>',
-                            '</span>',
+                        '<select class="' + replaced[0].className + ' is-replaced" id="' + replaced[0].id +
+                        '" name="' + replaced[0].name + '" ' + disabled + '>' +
+                        replaced[0].innerHTML +
+                        '</select>',
+                        '<span class="' + variables.customClass + ' ' + statuses.join(' ') + '" aria-hidden="true">',
+                        '<span class="' + variables.valueClass + '">' +
+                            CustomSelectReplacement.setInitialText($el) +
+                        '</span>',
+                        '<span class="icon-fallback-glyph">',
+                        '<span class="icon ' + variables.iconClass + '" aria-hidden="true"></span>',
+                        '<span class="text">Down arrow</span>',
+                        '</span>',
+                        '</span>',
```

Comment:
Oof. I'd just do /* eslint-enable indent, max-len */ then 😂

Writing HTML in JS is an iffy practice we do all over the codebase and should work on rooting out - the real fix here would be to put this HTML into an underscore template. But that doesn't need to be done in this PR.